{{cookiecutter.repostory_name}}/envs/prod/docker-compose.yml

```diff
@@ -185,7 +185,53 @@ services:
     restart: unless-stopped
     logging:
       <<: *logging
+  {% endif %}
 
+  {% if cookiecutter.log_aggregating == 'y' %}
+  promtail:
+    image: grafana/promtail:2.9.2
+    restart: unless-stopped
+    environment:
+      - LOKI_URL=${LOKI_URL}
+      - LOKI_USER=${LOKI_USER}
+      - LOKI_PASSWORD=${LOKI_PASSWORD}
+      - LOKI_CLIENT=${LOKI_CLIENT}
+      - LOKI_CLIENT_SERVER_GROUP=${LOKI_CLIENT_SERVER_GROUP}
+    volumes:
+      - /var/run/docker.sock:/var/run/docker.sock:ro
+    entrypoint:
+      - sh
+      - -euc
+      - |
+        cat <<EOF > /etc/promtail/config.yml
+        server:
+          http_listen_port: 9080
+          grpc_listen_port: 0
+        positions:
+          filename: /tmp/positions.yaml
+        clients:
+          - url: "${LOKI_URL}/api/prom/push"
+            basic_auth:
+              username: "${LOKI_USER}"
+              password: "${LOKI_PASSWORD}"
+            external_labels:
+              client: "${LOKI_CLIENT}"
+              client_server_group: "${LOKI_CLIENT_SERVER_GROUP}"
+        scrape_configs:
+          - job_name: containerlogs
+            docker_sd_configs:
+              - host: unix:///var/run/docker.sock
+                refresh_interval: 5s
+            relabel_configs:
+              - source_labels: ["__meta_docker_container_name"]
+                regex: "/(.*)"
+                target_label: "container"
+              - source_labels: ["__meta_docker_container_log_stream"]
+                target_label: "logstream"
+        EOF
+        /usr/bin/promtail -config.file=/etc/promtail/config.yml
```

Comment:
since we are already mounting single files, lets mount entrypoint script as well; 
1) docker-compose won't be that big :P
2) we can edit it properly with help of IDE

Reply:
I agree, but :D so I can either crate/delete it during setup-xxx.sh or do similar to nginx - create `promtail` directory in root dir and use it only for prod.

Reply:
`app/envs/prod/promtail-entrypoint.sh` sounds good

Reply:
`envs/prod/promtail.yml`